.github/workflows/deploy-1-setup.yml

```
@@ -268,19 +274,25 @@ jobs:
         id: version-modified
         run: |
           if [[ "${{ steps.base.outputs.root-spec-ref }}" == "${{ steps.current.outputs.root-spec-ref }}" ]]; then
-              echo "::warning::The version string hasn't been modified in this PR, but needs to be before merging."
+              echo "::warning::The version string in ${{ inputs.spack-manifest-path }} hasn't been modified in this PR, but needs to be before merging."
               exit 1
           fi
 
       - name: Same Model Version Failure Notifier
         if: failure() && steps.version-modified.outcome == 'failure'
+        # TODO: We don't yet support bumping files outside of ./spack.yaml, so this is a temporary solution.
+        env:
+          BUMP_COMMENT_BODY: |-
+            Alternatively, comment the following to have it updated and committed automatically:
+              * `!bump major` for feature releases
+              * `!bump minor` for bugfixes
         uses: access-nri/actions/.github/actions/pr-comment@main
         with:
           comment: |
               The model version in the `${{ inputs.spack-manifest-path }}` has not been updated.
-              Either update it manually, or comment the following to have it updated and committed automatically:
-              * `!bump major` for feature releases
-              * `!bump minor` for bugfixes
+              You can update it manually.
+              ${{ inputs.spack-manifest-path != './spack.yaml' && env.BUMP_COMMENT_BODY || '' }}
+
 
       - name: Check Projections - Specs
         # This step checks that the root spec has an appropriate projection in the spack.yaml file.
```

```suggestion
        # This step checks that the root spec has an appropriate projection in the manifest file.
```
If it can be any name probably best to use a generic description. See also L216, L315.